agents/PAW-03B Impl Reviewer.agent.md

```diff
@@ -207,16 +211,40 @@ For final PRs, load context from all phases in ImplementationPlan.md, Spec.md fo
    - Set strategy variable: `<prs or local>`
 
    **Step 7.2a: IF Review Strategy = 'prs' - Push Phase Branch and Create Phase PR**:
+   - **Git Commands Only**: Use `git push` exclusively - do NOT use GitHub MCP direct-push tools (preserves git history)
    - Verify on phase branch: `git branch --show-current` should show `<target>_phase[N]`
    - Push phase branch: `git push -u <remote> <target>_phase[N]`
-   - **REQUIRED**: Create Phase PR:
-     - **PR Operations Context**: Provide branch names (source: phase branch, target: Target Branch), Work Title, Issue URL
-     - Source: `<target>_phase[N]` → Target: `<target_branch>`
-     - Title: `[<Work Title>] Implementation Phase <N>: <brief description>`
-     - Include phase objectives, changes made, testing performed
-     - Link to Issue URL from WorkflowContext.md
-     - Artifact links: Implementation Plan at `.paw/work/<feature-slug>/ImplementationPlan.md`
-     - At bottom: `🐾 Generated with [PAW](https://github.com/lossyrob/phased-agent-workflow)`
+- **REQUIRED**: Create Phase PR:
+  - **PR Operations Context**: Provide branch names (source: phase branch, target: Target Branch), Work Title, Issue URL
+  - Source: `<target>_phase[N]` → Target: `<target_branch>`
+  - Title: `[<Work Title>] Phase <N>: <brief description>` (emphasize "Phase N" to clearly distinguish from final PR)
+  - Body format:
+    ```
+    Related to #<N>
+    
+    **This is Phase <N> of a phased implementation approach.** This PR represents an intermediate checkpoint in the implementation process, not the final deliverable.
+    
+    ### Phase Objectives
+    [What this phase accomplishes]
+    
+    ### Changes Made
+    [Summary of implementation]
+    
+    ### Testing Performed
+    [Verification results]
+    
+    ### Artifacts
+    - Implementation Plan: `.paw/work/<feature-slug>/ImplementationPlan.md`
+    
+    ---
+    🐾 Generated with [PAW](https://github.com/lossyrob/phased-agent-workflow)
+    ```
+  - Extract issue number from WorkflowContext.md's Issue URL field (same pattern as Planning PR):
```

Comment:
Don't mention planning PR. And workflow context comes from tool call as mentioned elsewhere.